### PR TITLE
Dashboard Issues

### PR DESCRIPTION
**Pull request by MrFlashAccount**
### Pull Request Description

Closes: https://github.com/enso-org/cloud-v2/issues/1592

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.
- [ ] If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.
  - If applicable, it is suggested to paste a link to a successful run of the Extra Tests.

### CODE REVIEW

File: app/gui/index.html [hunk 1]
```diff
@@ -38,6 +38,15 @@
...
     />
     <title>Enso %ENSO_IDE_VERSION%</title>
+
+    <script type="module">
+      import { platform } from  '../common/src/detect.ts'
+
+      const currentPlatform = platform()
+
+      document.documentElement.setAttribute('data-version', '%ENSO_IDE_VERSION%')
+      document.documentElement.classList.add(currentPlatform.toLowerCase())
```
Comment (somebody1234) [inline app/gui/index.html:48]: scuff

can we prefix the platform name please? so like `platform-macos`

File: app/gui/tailwind.config.js [hunk 2]
```diff
@@ -657,6 +657,10 @@ inset 0 -36px 51px -51px #00000014`,
 
       addVariant('not-focus', '&:where([data-rac]):not([data-focused])')
       addVariant('not-selected', '&:where([data-rac]):not([data-selected])')
+
+      addVariant('windows', '.windows &')
+      addVariant('macos', '.macos &')
```
Comment (somebody1234) [inline app/gui/tailwind.config.js:662]: if applying the prefix suggestion above then we should do it here too:
```js
      addVariant('platform-macos', 'html.platform-macos &')
```

**Review by somebody1234 — APPROVED** [review]
CR :white_check_mark:

File: app/gui/index.html (hunk 1 above)
Comment (MrFlashAccount) [reply app/gui/index.html:48]: Why?

Comment (somebody1234) [reply app/gui/index.html:48]: i guess mostly to indicate that they are variants of a single property (meaning that there shouldn't be more than one of `platform-` on any element)

Comment (MrFlashAccount) [reply app/gui/index.html:48]: Not sure what you mean. I can specify `macos:text-12 windows:text-24 linux:text-10`. Yes, they exclude each other, but i don't understand how `platform` relates to it

Comment (somebody1234) [reply app/gui/index.html:48]: oh true :skull: this is a modifier/variant, not a property

but i think regardless it still makes sense to prefix the classes/variants just to make it clear that they are related to each other. just my two cents

also any prefix works, it doesn't have to be platform

**Review by PabloBuchu — APPROVED** [review]

**Pull request merged**